pypi-setuptools.yaml

```diff
@@ -0,0 +1,56 @@
+package:
+  name: pypi-setuptools
+  version: 69.2.0
+  epoch: 1
+  description: Easily download, build, install, upgrade, and uninstall Python packages
+  copyright:
+    - license: "MIT"
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - python-3.10-base
+      - python-3.11-base
+      - python-3.12-base
+      - wolfi-base
+  environment:
+    # This is needed to work around the error "ValueError: ZIP does not support timestamps before 1980"
+    SOURCE_DATE_EPOCH: 315532800
+
+vars:
+  pypi-package: setuptools
+
+data:
+  - name: py-versions
+    items:
+      3.12: ""
+      3.11: ""
+      3.10: ""
+
+pipeline:
```

Comment:
This means the package will be empty, which might be surprising to folks. We have a linter that might complain about it too, which is not blocking.

Is it possible this package being empty is causing the problems you're seeing in CI? Does adding something to `${{targets.destdir}}/blah` cause anything different to happen?

Reply:
let me try that, i do see that "empty" build base does do `mkdir ${{targets.destdir}}` and that it ends up containing an sbom......

.... maybe our sbom generator fails without even a destdir?

Reply:
i see curl rustlts does something similar and explicitely marks the parent package as empty.